.ci/windows-steps.yaml

```diff
@@ -60,8 +60,8 @@ steps:
 
     cmake $(CMakeGenerator) `
     $(CMakeArgs) `
-    -DBLAS_LIBRARY:FILEPATH=$(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\lib\x64\libopenblas.dll.a `
-    -DLAPACK_LIBRARY:FILEPATH=$(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\lib\x64\libopenblas.dll.a `
+    -DBLAS_LIBRARIES:FILEPATH=$(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\lib\x64\libopenblas.dll.a `
```

Comment:
We can probably avoid having to change this flag by adding a case to map X_LIBRARY to X_LIBRARIES for the impacted libraries. Does this seem like a good idea?

Reply:
Is there a specific reason for using such an ancient version (0.2.14) of OpenBLAS ?  

OpenBLAS has had releases with many bug fixes since then:  
https://github.com/xianyi/OpenBLAS/releases

Reply:
That's true, but the pipeline is currently using nuget, and the latest version [there](https://www.nuget.org/packages/OpenBLAS/) seems to be 0.2.14.1
Though there seem to be other openblas nuget projects, so we could try switching to those to get all the way to [0.2.19.3](https://www.nuget.org/packages/OpenBLASLibs/).
I think it makes more sense to consider another windows package manager such as vcpkg, as [that openblas](https://github.com/microsoft/vcpkg/tree/master/ports/openblas) seems to be up to date. Though that's probably a different PR.

Reply:
Well, we can certainly trust the Windows "world" to be a royal mess in many aspects. Instead of jumping through these silly hoops, why not just simply ship a pre-compiled copy of OpenBLAS along with mlpack?

Reply:
I think actually that's exactly what we're doing---we're just grabbing the version from nuget.  I have no issue switching versions or anything like that... if you want to try that feel free, either in this PR or another or wherever.  But yeah... the Windows distribution world is a total nightmare...